build.gradle

```diff
@@ -29,11 +29,11 @@ test {
 }
 
 application {
-    mainClass = "seedu.duke.Duke"
+    mainClass = "seedu.duke.WellNus"
```

Comment:
Hmm, room for further future personalisation by putting our team number into the package. But problem for another day!